Formula/nfdump.rb

```diff
@@ -11,8 +11,16 @@ class Nfdump < Formula
     sha256 "5bbd8ec9a51cde3c6ed0aaeb1b77a4f2ba59bea99d5304fa1415a7e2e0d5280b" => :mountain_lion
   end
 
+  option "with-readpcap", "Read netflow packets from a give pcap_file instead of the network. Is intended for debugging only."
```

Comment:
I think that it's important to keep the "debugging only" bit since the developer mentions it, but I can pull it out if you think that brevity is more important.